lib/prompt_template.ex

```diff
@@ -339,9 +393,15 @@ defmodule LangChain.PromptTemplate do
           [Message.t()] | no_return()
   def to_messages!(prompts, %{} = inputs \\ %{}) when is_list(prompts) do
     Enum.map(prompts, fn
-      %PromptTemplate{} = template ->
+      %PromptTemplate{content_type: content_type} = template when content_type in [nil, :text] ->
         to_message!(template, inputs)
 
+      %PromptTemplate{content_type: content_type} = template
+      when content_type in [:image, :file] ->
+        # For image/file templates, create a user message with the content part
```

Comment:
Here I'm not sure if it makes sense to have images/files that don't have the user role